docs/integration_resources/software/resolve.md

```diff
@@ -17,13 +17,13 @@ Apart from the official requirements, the `PYTHONHOME` environment is required t
 
 To use the Python interpreter provided with Shift, set up the variable in this way:
 
-**PYTHONHOME** : <path_to_your_shift_folder>/shift/thirdparty/python
+**PYTHONHOME** : <path_to_your_shift_folder&gt/shift/thirdparty/python

```

Comment:
With the change, on my side I have this result:
![image](https://github.com/user-attachments/assets/94a02444-2c0f-4eb0-982e-0bfe7041d21d)

Reply:
Done